### PR TITLE
Refactor util functions

### PR DESCRIPTION
**Pull request by spectranaut**
Hi @mcking65 -- I just did a minor refactored the util functions for ease of use. Unfortunately this mean updating a couple files that are still in review. Can you not merge until #783 and #784? I might have to rebase if anything in those two PRs need updating before merge.

Perhaps @jessebeach can do a review on this one :)

The relevant commits to review start at "Change util names to assert" -- the rest are the commits from #784

### CODE REVIEW

File: test/util/assertRovingTabindex.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  // tabindex='0' is expected on the first element
+  let elements = await t.context.session.findElements(By.css(elementsSelector));
+
+  // test only one element has tabindex="0"
+  for (let tabableEl = 0; tabableEl < elements.length; tabableEl++) {
+    for (let el = 0; el < elements.length; el++) {
+
+      let tabindex = el === tabableEl ? '0' : '-1';
+
+      assert.deepEqual(
```
Comment (jessebeach) [inline test/util/assertRovingTabindex.js]: You shouldn't need `deepEqual` to assert the equivalence of integers or strings.

File: test/util/assertRovingTabindex.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+
+/**
+ * Confirm the roving tabindex has been initialized for a list of elements.
+ *
+ * @param {obj} t                   - ava execution object
+ * @param {String} elementsSelector - selector for elements which have roving tabindex
+                                      by default, focus should be on the first item
+ * @param {webdriver.Key} key       - which key to change roving focus between items
```
Comment (jessebeach) [inline test/util/assertRovingTabindex.js]: In a perfect world, we'd make this an enum.

**Comment by spectranaut** [comment]
I'm going to rebase this PR then it should be good for merge now that #784 is in

**Comment by spectranaut** [comment]
Okay this should be good to go, @sh0ji! Addressed the "deepEqual" point too, @jessebeach

**Comment by jessebeach** [comment]
LGTM, let's get it merged.

**Pull request merged**